test/contrib/bigquery_test.py

```diff
@@ -38,8 +38,7 @@
 
 @attr('gcloud')
 class TestLoadTask(bigquery.BigqueryLoadTask):
-    client = None
-
+    client = MagicMock()
```

Comment:
To test locally (Will add this to docs):

* Download and install the gcloud tool from google.
* Login to set your default app credentials using `gcloud auth login`
* Ensure you have access to the test project in GCS
* `GCS_TEST_PROJECT_ID=luigi-travistestenvironment ./venv/bin/tox -e gcloud -- test/contrib/bigquery_test.py:BigqueryTest`